docs/modules/develop/pages/releases.adoc

```diff
@@ -227,6 +227,45 @@ That way they don't synchronize with Github.
 At the top you should have an `Unreleased` header with the `Added`, `Changed`, `Fixed` and `Removed` empty sections.
 After that, the header with the current version.
 Add the `Unreleased` section or create the new current version section.
+. Update the version in the documentation system that we use, Antora. As its multi repository you'll need to do it on both places:
+.. in the https://github.com/decidim/decidim[decidim repository]. This is done in https://github.com/decidim/documentation/blob/master/antora-playbook.yml[docs/antora.yml], by changing the version key, from `version: develop` to `version: v0.XX` (for instance `version: v0.27`).
```

Comment:
I think this part we could automate during the `update_versions` rake task.

E.g. within `Decidim::GemManager`:

```ruby
  def replace_versions
    # ...

    antora_conf = File.join(root, "docs/antora.yml")
    docs_version = version
    docs_version = "develop" if docs_version.match?(/\.dev$/)
    File.write(antora_conf, File.read(antora_conf).sub(/^version: .+/, "version: #{docs_version}"))
  end
```

What do you think?